### ISSUE TITLE
Submit a session proposal for the World Biodiversity Forum 2024

### ISSUE THREAD

**Issue by Daniel-Mietchen** [Daniel-Mietchen/ideas#1785]
as per
https://worldbiodiversityforum2024.org/abstract-submission/ .

**Comment by Daniel-Mietchen** [comment]
They had rather strict limits in that a maximum of two submissions were permitted per person - one each for a talk or a poster.

I ended up filling my slots with the following entries:
- Session: MONMOD_10.2: Navigating the social and political dimensions of biodiversity data
- Submission type: oral presentation
- Submission title: WIKIMEDIA PROJECTS AS A REFLECTION OF HUMAN DIMENSIONS OF BIODIVERSITY DATA
- Submission abstract: The Wikimedia ecosystem of Wikipedia, Wikidata, Wikispecies and their sister sites contains a rich body of information that reflects various human dimensions of biodiversity data. This contribution will provide a guided tour through this ecosystem and highlight how its study could assist in improving the understanding of human dimensions of biodiversity data. The types of questions for the tour to touch upon is broad: Which facets of biodiversity are covered in detail or not at all? Which taxonomic groups, biogeographic ranges or socio-economic factors are over- or under-represented in which parts of the ecosystem as compared to pertinent scholarly publications, policy documents or mass media from past and present? How often are biodiversity-related entries viewed or edited in different parts of the Wikimedia ecosystem? In which languages do Wikimedia projects provide information about biodiversity-related topics closely associated with a well-circumscribed geographic or cultural entity? How are human contributions to biodiversity loss covered, or policies aimed at biodiversity preservation? How do Wikimedia communities handle biases? Combined with its multiple dimensions of openness - e.g. openness towards contributions from anyone, open licensing that encourages reuse, adherence to open standards, and large-scale integration with other biodiversity data resources - the Wikimedia ecosystem can thus provide a fertile ground for studying human dimensions of biodiversity data.

- Session: CUL_8.3: Models and scenarios for biodiversity & NCP at regional to global scales
- Submission type: poster
- Submission title: A MATHEMATICAL PERSPECTIVE ON THE SHARING OF BIODIVERSITY-RELATED MODELS
- Submission abstract: Biodiversity-related sciences - from metabolomics to traits to invasions to systems biology and ecosystem dynamics - are among a broad range of research fields that make increasing use of mathematical and computational models. There is, however, room for improvement in terms of how models and scenarios for biodiversity and NCP are being shared, both within the field and with other fields, including mathematical and computational ones. This contribution will outline challenges, opportunities and common practices in the sharing of mathematical and computational models and highlight how they translate to biodiversity and NCP contexts at different biogeographic scales, levels of biological organization or socio-economic factors. Special emphasis will be put on the discoverability, reproducibility and reusability of biodiversity-related models and on how the sharing of such models can both benefit from and contribute to research in mathematical, computational and other fields.

The acronym NCP refers to Nature's contributions to people, and since it was defined as such in the session's description, I did not define it again in my submission.
